kernel_gateway/tests/test_jupyter_websocket.py

```diff
@@ -150,6 +150,20 @@ def _set_api():
         self.assertRaises(ImportError, _set_api)
 
     @gen_test
+    def test_options_and_auth_token(self):
+        """OPTIONS requests should not need to submit a token."""
+        # Set token requirement
+        app = self.get_app()
```

Comment:
@parente: The docstring of [`get_app`](https://github.com/SimonBiggs/kernel_gateway/blob/03c53b4423f7fafccefbc259501405c60a438d83/kernel_gateway/tests/test_gatewayapp.py#L90) talks about a "Tornado test runner". Are we using that? If not, could that be the reason why `self.request` wasn't set? I haven't seen the original error message, I'm just second-guessing from the conversation.